lib/src/components/media-card.ts

```diff
@@ -0,0 +1,80 @@
+import '../globals';
+
+const { LitElement, html, css } = window.Lit;
+
+class MediaCard extends LitElement {
+    static styles = css`
+        
+        .wrapper {
+            display: flex;
+            justify-content: center;
+            flex-wrap: wrap;
+            margin-top: 6.25rem;
```

Comment:
What is this margin-top value for?

Reply:
It was the top margin for the wrapper, which was not needed so I removed it.